app/assets/stylesheets/govuk-component/_task-list.scss

```diff
@@ -50,24 +50,26 @@ $dot-size: 16px;
   }
 
   &--title {
-    @include bold-24;
+    @include _core-font-generator(19px, 19px, 19px, 1.4, 1.4, false, bold);
```

Comment:
You can pass options through to the normal mixins too e.g. `@include bold-19(a, b, c)`